SharePointFramework/PortfolioWebParts/src/components/ProjectTimeline/index.tsx

```diff
@@ -289,10 +320,48 @@ export class ProjectTimeline extends Component<IProjectTimelineProps, IProjectTi
         itemProps: { style },
         project: project.title,
         projectUrl: project.url,
-        phase: project.phase
+        phase: project.phase,
+        type: project.type,
+        budgetTotal: project.budgetTotal,
+        costsTotal: project.costsTotal
+      } as ITimelineItem
+    })
+
+    const phases: ITimelineItem[] = timelineItems.filter((item) => item.type !== strings.ProjectLabel).map((item, id) => {
+      id += items.length
+
+      const backgroundColor = item.type === strings.PhaseLabel
+        ? '#2589d6'
+        : item.type === strings.MilestoneLabel
+          ? 'transparent'
+          : item.type === strings.SubPhaseLabel
+            ? '#249ea0'
+            : '#484848'
+
+      const group = _.find(groups, (grp) => item.title.indexOf(grp.title) !== -1)
```

Comment:
This function seems a bit complex. Can anything be simplified? Open question <3

Reply:
Best I can think of as of now would be:
`const group = _.find(groups, (grp) => project.title.includes(grp.title))`

Not sure if we should start using ".includes" as it doesn't have Internet Explorer support though. ([source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes#browser_compatibility))

Reply:
Underscore has `contains` 🚀